pytest_djangoapp/fixtures/settings.py

```diff
@@ -4,10 +4,21 @@
 import pytest
 from django.conf import settings as django_settings
 
+from pytest_djangoapp.configuration import Configuration
 
 _UNSET = set()
 
 
+@pytest.fixture(scope="session")
+def djapp_options():
+    yield Configuration.get()[Configuration._prefix]
+
+
+@pytest.fixture(scope="session")
+def app_name(djapp_options):
```

Comment:
We should probably call it `conf_app_name` to add more context. So we could add to `conf_` namespace later if needed.

Please add a docstring. 
Also updating the docs would be nice.